elfi/examples/gauss.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import scipy.stats as ss
+import elfi
+from functools import partial
+
+
+"""An example implementation of a Gaussian noise model.
+
+The design of the example is identical to ma2.py
+"""
+
+
+def Gauss(mu, sigma, n_obs=20, batch_size=1, random_state=None):
+    # Standardising the parameter's format.
+    mu = np.asanyarray(mu).reshape((-1, 1))
+    sigma = np.asanyarray(sigma).reshape((-1, 1))
+    y = ss.norm.rvs(loc=mu, scale=sigma, size=(batch_size, n_obs),
+        random_state=random_state)
+    return y
+
+
+def ss_mean(x):
+    """The summary statistic corresponding to the mean.
+    """
+    ss = np.mean(x, axis=1)
+    return ss
+
+
+def ss_var(x):
+    """The summary statistic corresponding to the variance.
+    """
+    ss = np.var(x, axis=1)
+    return ss
+
+
+def get_model(n_obs=20, true_params=None, seed_obs=None):
+    """Returns a complete Gaussian noise model
+
+    Parameters
+    ----------
+    n_obs : int
+        the number of observations
+    true_params : list
+        true_params[0] corresponds to the mean,
+        true_params[1] corresponds to the standard deviation
+    seed_obs : None, int
+        seed for the observed data generation
+
+    Returns
+    -------
+    m : elfi.ElfiModel
+    """
+
+    if true_params is None:
+        true_params = [10, 2]
+
+    y_obs = Gauss(*true_params, n_obs=n_obs,
+        random_state=np.random.RandomState(seed_obs))
+    sim_fn = partial(Gauss, n_obs=n_obs)
+
+    m = elfi.ElfiModel(set_current=False)
+    elfi.Prior('norm', -1e0, 1e3, model=m, name='mu')
```

Comment:
That's pretty wide: variance of one million. :D Now most of the simulations will be run with completely irrelevant values. How about a uniform prior for `mu` and a truncated (positive) norm for sigma?